audits/viewport-meta-tag/index.js

```diff
@@ -28,22 +28,41 @@ class ViewportMetaTagTest {
       return Promise.reject('No data provided.');
     }
 
-    if (typeof inputs.dom !== 'function') {
-      return Promise.reject('No DOM Parser provided.');
+    if (typeof inputs.driver === 'undefined') {
```

Comment:
this is inherited from all the tests, but there doesn't seem to be a need to be this defensive. If `inputs` or `inputs.driver` are undefined, the test will fail quickly and obviously :)